TemporaryOysters/bangazon_website
# Added product types

##Pull Requests



##Descriptions:
I added in the product type model and the product type test.
* Please describe what issue this PR solves











##Steps to Test:
-git checkout master
-git fetch
-git checkout rvs-inital
-$python manage.py makemigrations bangazon_webstore
-python manage.py migrate
-$python manage.py test bangazon_webstore
-7 tests should pass


##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?


* Any command line utilities to run?
